ipti/br.tag
# feature: sedsp add classroom with class code

## Added functionality to add classes in the SEDSP module
#### Description
This change adds new functionality to the SEDSP module, allowing users to add a class through a specific code, optionally including enrollments and students.

#### Added functionality
When accessing the SEDSP module, there is now an option to add a new class through a specific code. Additionally, users can check a checkbox to include enrollments and students related to that class.

#### Motivation
The motivation for this change came from feedback received from users of the SEDSP module, who reported the need for a more efficient way to add classes and their students to the system.

#### How to test
To test this functionality, follow these steps:
- Access the SEDSP module.
- Click the "Add Class" button.
- Enter the desired class code.
- Optionally, check the checkbox to include enrollments and students.
- Click "Save" to finish adding the class.
> Make sure that the class and its students have been correctly added to the system.
